### PR TITLE
Advantage shop confirmation

### PR DESCRIPTION
**Pull request by sowasred2012**
# Done

Added information regarding a new subscription to /advantage

# QA
- Visit /advantage/subscribe, and sign in with SSO if you are prompted to
- Make your product selection, add it to the cart and click "Buy now"
- Fill out the payment details
  - Fill out all card fields by tapping the 4 & 2 keys until you can't input anything any more
    - (you'll end up with card number: 4242424242424242, expiry: 04/24, cvc: 242 and zip: 42424)
  - Address fields can be anything
- Click "Continue", accept Ts&Cs, then click "Pay"
- The payment should be successful, and when it completes you'll be redirected to /advantage
- Your new subscription should be easily visible on the page, with a "New" badge, the "Attached" tab should be expanded and include a welcome message along with instruction on attaching your machines.
- There should also be a "Where?" button (in reference to configuring services), and clicking that should trigger an animated icon on tabs for the services your subscription includes.

### CODE REVIEW

File: templates/advantage/table/_contract-name.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          {% elif contract["contractInfo"]["daysTillExpiry"] == 1 %}
+            Ends in 1 day
+          {% else %}
+            Ends in {{ contract["contractInfo"]["daysTillExpiry"] }} days
+          {% endif %}
+        </div>
+      {% endif %}
+    {% endif %}
+  </button>
+</td>
```
Comment (carkod) [inline templates/advantage/table/_contract-name.html:25]: new line?

File: templates/advantage/table/_expiry-date.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <tr style="border-top: 0;">
+    <td class="u-align--right">Ends:</td>
+    <td class="u-align--left">
+      <strong>{{ contract['contractInfo']['effectiveToFormatted'] }}</strong>
+      {% if contract["renewal"]["recently_renewed"] %}
+      <div class="p-label--updated">Renewed</div>
+      {% endif %}
+    </td>
+  </tr>
+{% endif %}
```
Comment (carkod) [inline templates/advantage/table/_expiry-date.html:34]: new line?

**Review by steverydz — CHANGES_REQUESTED** [review]
Looks good. Just one suggestion in the code and there's a couple of new lines missing at the end of some files.

**Review by steverydz — APPROVED** [review]
LGTM :+1:

**Pull request merged**